packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -23,6 +22,8 @@ dependencies:
   path_provider_platform_interface: ^1.0.1
   path_provider_macos: ^0.0.4
   path_provider_linux: ^0.0.1
+  path_provider_windows:
+    path: ../path_provider_windows
```

Comment:
This can't be landed this way. Normally I would say you'd need to remove this and the endorsement, and land those as a follow-up once path_provider_windows was landed and published, but since it's already been published you should be able to just changes this to ^0.0.1 now.